kalite/distributed/management/commands/retrievecontentpack.py

```diff
@@ -176,29 +174,18 @@ def process_content_pack(self, zf, lang):
 
 
 def extract_content_pack_metadata(zf, lang):
+    lang = lcode_to_django_lang(lang)
+    modir = get_po_filepath(lang)
+    if not os.path.exists(modir):
+        os.makedirs(modir)
```

Comment:
The `modir` is no longer relevant for content packs, so I'm not sure what this is doing here? The variable is not even used :)